Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1168,7 +1168,7 @@ LayoutUnit GridTrackSizingAlgorithmStrategy::minContributionForGridItem(RenderBo
         return minContentContributionForGridItem(gridItem, gridLayoutState);
 
     auto& gridItemMinSize = isRowAxis ? gridItem.style().logicalMinWidth() : gridItem.style().logicalMinHeight();
-    bool overflowIsVisible = isRowAxis ? gridItem.effectiveOverflowInlineDirection() == Overflow::Visible : gridItem.effectiveOverflowBlockDirection() == Overflow::Visible;
+    bool overflowIsVisible = isRowAxis ? (gridItem.effectiveOverflowInlineDirection() == Overflow::Visible || gridItem.effectiveOverflowInlineDirection() == Overflow::Clip) : gridItem.effectiveOverflowBlockDirection() == Overflow::Visible;
```

Comment:
I think it would be nice if we didn't have to potentially call `effectiveOverflowInlineDirection()` twice so maybe we should store the overflow value in a local variable here. 

Also `overflowIsVisible` should probably be renamed since that's not what we're looking for anymore. The spec says "its computed overflow is not a scrollable overflow value" so we should probably try to make it match in some way